cmd/kube-apiserver/app/server.go

```diff
@@ -151,8 +151,6 @@ func NewAPIServer() *APIServer {
 
 // NewAPIServerCommand creates a *cobra.Command object with default parameters
 func NewAPIServerCommand() *cobra.Command {
-	s := NewAPIServer()
-	s.AddFlags(pflag.CommandLine)
```

Comment:
I think it has the effect of binding all the server flags to the command line. This has side effects.